src/parachain/transaction.ts

```diff
@@ -29,10 +33,27 @@ export interface TransactionAPI {
         extrinsics: SubmittableExtrinsic<"promise", ISubmittableResult>[],
         atomic?: boolean
     ): SubmittableExtrinsic<"promise", ISubmittableResult>;
+
+    /**
+     * Getter for fee estimation of the extrinsic.
+     *
+     * @param {SubmittableExtrinsic<"promise">} extrinsic Extrinsic to get fee estimation about.
+     * @returns {MonetaryAmount<Currency>} amount of native currency that will be paid as transaction fee.
+     * @note This fee estimation does not include tip.
+     */
+    getFeeEstimation(extrinsic: SubmittableExtrinsic<"promise">): Promise<MonetaryAmount<Currency>>;
```

Comment:
nit: I'd probably just call it `getFeeEstimate`
"Estimation" is the process of finding an estimate.

Reply:
Ah, nice catch, changed)